R/paste_skip_na.R

```diff
@@ -0,0 +1,64 @@
+#' Concatenate strings dropping missing values
+#'
+#' @details If all values are missing, the value from the first argument is
+#'   preserved.
+#'
+#' @param ...,sep,collapse See \code{?paste}
+#' @return A character vector of pasted values.
+#' @examples
+#' paste_skip_na(NA) # NA_character_
+#' paste_skip_na("A", NA) # "A"
+#' paste_skip_na("A", NA, c(NA, "B"), sep = ",") # c("A", "A,B")
+#' @export
+paste_skip_na <- function(..., sep=" ", collapse=NULL) {
+  args <- list(...)
+  if (length(args) <= 1) {
+    if (length(args) == 0) {
+      # match the behavior of paste
+      paste(sep=sep, collapse=collapse)
```

Comment:
Nit: could you add spaces like `sep = sep` as you have in the example on line 11?

Reply:
I intend to change my coding style to that, but it's still not in my muscle memory.  😄